website/pages/vagrant-cloud/boxes/using.mdx

```diff
@@ -10,7 +10,7 @@ A primary use case of Vagrant Cloud by HashiCorp is to be able to easily find
 boxes you can use with Vagrant that contain the technologies you need
 for a Vagrant environment. We've made it extremely easy to do that:
 
-1. Go to the [Discover page](https://vagrantcloud.com/discover), and search for
+1. Go to the [Discover page](https://vagrantcloud.com/), and search for
```

Comment:
Good catch! 
I think this should go to the search page. In particular if the user already has an account and is logged in https://vagrantcloud.com/ will go to that users dashboard.

```suggestion
1. Go to the [Discover page](https://app.vagrantup.com/boxes/search), and search for
```